src/api.ts

```diff
@@ -15,7 +15,7 @@ export const HEADERS = {
 //* USER AUTHENTICATION
 const BASE_AUTH = `${BASE_URL}/Users`;
```

Comment:
We don't need to add the base url again as it is included in the axios instance.

Reply:
done